protocol-definitions/Map.yaml

```diff
@@ -2965,4 +2965,26 @@ methods:
           since: 2.4
           doc: |
             the function to apply to each entry.
-    response: {}
\ No newline at end of file
+    response: {}
+  - id: 73
+    name: putAllWithMetadata
+    since: 2.7
+    doc: |
+      Put a batch of entries with metadata into an IMap. This is a client side version of WAN entry replication.
```

Comment:
may be you can add some more details as in the doc of putAll message (includes some details about partitions etc.).